### PR TITLE
Fix Outlook p margin bug, set margin to default on anchors inside p

### LINKED ISSUE #421: CSS reset on hyperlinks causes paragraph spacing issue (Outlook)
**How can we reproduce this bug?**

Within a column, add two paragraphs - the first containing a link. E.g.

```
<columns>
<p>If you did not request this change please<a href="#">contact us</a>.</p>
<p>Best regards.</p>
</columns>
```

**What did you expect to happen?**

The paragraphs to have the normal, Foundation-defined margin between them.

**What happened instead?**

The first paragraph has no margin-bottom, so the paragraphs run against each other as if there was only a linebreak between them. This appears to be caused by a CSS reset within the Foundation framework which adds `Margin: 0; margin:0;` to all 'a' elements - this overrules the 'p' element's margin.

It's possible to remedy it by adding custom CSS but I'm not sure whether this is an adequate solution?

`p a { Margin: default; margin: default; }`

Here's a screenshot demonstrating what happens:

image

**What email clients does this happen in?**

Outlook 2007, 2010, 2013 and 2016.

### PR DESCRIPTION
**Pull request by jsit**
Fixes #421

### CODE REVIEW

**Comment by nedtwigg** [comment]
Thanks so much!  This ate a ton of my time - I had resorted to `<!--[if mso]> <br> <![endif]-->`, but this fix works great!

**Comment by jsit** [comment]
Can we get this merged?

File: scss/components/_typography.scss [hunk 1]
```diff
@@ -243,6 +243,11 @@ p {
     line-height: $subheader-lineheight;
     color: $subheader-color;
   }
+
+  a { // To fix Outlook margin issue #421
+    Margin: default;
+    margin: default;
```
Comment (DanielRuf) [inline scss/components/_typography.scss:248]: Can you switch the order of the rules? Is `inherit` not what we want to use here?

**Review by DanielRuf — CHANGES_REQUESTED** [review]

File: scss/components/_typography.scss (hunk 1 above)
Comment (jsit) [reply scss/components/_typography.scss:248]: Order switched. I think we want `default`; we wouldn't want to inherit any margins from a parent element. At least that's the value that's used in #421, where I found this fix.

**Pull request merged**